internal/pkg/cli/merger/consts.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2023 The Kubernetes Authors.
```

Comment:
nit
```suggestion
Copyright 2024 The Kubernetes Authors.
```

the same applies to other new files.